src/system/SystemPacketBuffer.h

```diff
@@ -637,6 +625,15 @@ class DLL_EXPORT PacketBufferHandle
         return PacketBufferHandle(buffer);
     }
 
+    void Free()
```

Comment:
So this is basically what `operator=` taking `nullptr` does, right?  It seems to me that we should have one API or the other but not both for saying "make this an empty handle"....

Reply:
Done and done.